cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,103 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+    cy.viewport(1920, 1080);
+  });
+
+  it('should add a new worker', () => {
+    cy.createWorker();
```

Comment:
Function cy.createWorker() is not defined in this file. Ensure it is either imported or defined in this file.